features/txt-parfmt-props.feature

```diff
@@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
   I need a ParagraphFormat object with read/write formatting properties
 
 
+  @wip
+  Scenario Outline: Get tab stops object
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops is a CT_TabStops object
+      
+    Examples:  values for paragraph_format.tab_stop elements 
+      | value |
+      | no    |
+      | some  |
+      
```

Comment:
Hi David, what I meant by "the implementation of this bit" was the unit-test/implementation pairing(s) that when complete allow the acceptance test I merged to pass.

Then the rest of these acceptance tests can probably come as a group; I'm assuming it compresses to four or so rather than the original nine.

It's a judgement call how many acceptance tests ahead you develop. There's certainly nothing wrong with doing them onesy-twosy at a time with implementation in-between. The problem with stacking up too many is you're getting ahead of your understanding of the implementation and raising the risk you need to go back and change something based on what you learn.

But in this case, four or so is okay, once the actual access of a TabStops object from ParagraphFormat is separated out. These would all be tests of the collection behaviors of TabStops, so it's a coherent set.

Let me know if you need more.